package.json

```diff
@@ -34,16 +34,18 @@
     "test:material": "web-test-runner --config web-test-runner-material.config.js",
     "test:snapshots": "web-test-runner --config web-test-runner-snapshots.config.js",
     "test:webkit": "web-test-runner --config web-test-runner-webkit.config.js",
+    "test:smoke": "web-test-runner --config web-test-runner-smoke.config.js",
     "update:lumo": "TEST_ENV=update web-test-runner --config web-test-runner-lumo.config.js",
     "update:material": "TEST_ENV=update web-test-runner --config web-test-runner-material.config.js",
     "update:snapshots": "web-test-runner --config web-test-runner-snapshots.config.js --update-snapshots"
   },
   "devDependencies": {
+    "@esm-bundle/chai": "^4.3.4-fix.0",
     "@fontsource/roboto": "^4.5.1",
     "@polymer/iron-component-page": "^4.0.0",
     "@typescript-eslint/eslint-plugin": "^4.31.2",
     "@typescript-eslint/parser": "^4.31.2",
-    "@vaadin/testing-helpers": "^0.3.0",
+    "@vaadin/testing-helpers": "^0.3.1",
```

Comment:
Added these to the root package.json in order to be able to run tests within `/test` (which is not an actual package). I'm not super confident with this approach, as I don't have a lot of experience with monorepos.

If we want to go with this approach we could remove these from the individual packages I guess.

Reply:
Yes, we can indeed remove them from individual packages in a separate PR.